src/main/java/com/codeup/maktrak/controllers/UserController.java

```diff
@@ -0,0 +1,39 @@
+package com.codeup.maktrak.controllers;
+
+
+import com.codeup.maktrak.daos.UserRepository;
+import com.codeup.maktrak.models.User;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.persistence.GeneratedValue;
+
+@Controller
+public class UserController {
+    private UserRepository usersRepository;
+    private PasswordEncoder encoder;
+
+    public UserController(UserRepository usersRepository, PasswordEncoder encoder) {
+        this.usersRepository = usersRepository;
+        this.encoder = encoder;
+    }
+
```

Comment:
Need A Path to Login User; right now there is no way for us to post the user login credentials yet. You need a `@PostMapping("/Login")` or something of the sorts here.

Reply:
There's no need for a `@PostMapping` when the security component is being used, that's one of the things that will be automatic.